magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -127,7 +129,12 @@ public class NavigationDelegate(
   ) {
     currentNavigable!!.transitionStarted()
     to?.whenMeasured {
-      magellanTransition.animate(from, to, direction) {
+      val transition = if (shouldRunAnimations()) {
+        magellanTransition
+      } else {
+        NoAnimationTransition()
+      }
+      transition.animate(from, to, direction) {
```

Comment:
Hmm, the only thing is that now all custom navigators will have to check this same field to know whether to run animations. Not necessarily a problem, but worth noting that we're making the Navigator contract more complex

Reply:
Thats fair. We can update the contract (interface) to denote that every consumer should probably support this.